aspnetcore/mvc/razor-pages/razor-pages-authorization-conventions/sample/Pages/_ValidationScriptsPartial.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿<!--
+    Don't use the ajax.aspnetcdn.com CDN in production.
+-->
```

Comment:
Consider using a Razor comment here instead, to prevent the comment from displaying in the rendered markup.